### ISSUE TITLE
Pycharm crashes badly when I select a background with opacity

### ISSUE THREAD

**Issue by Fr-S** [ChrisRM/material-theme-jetbrains#1230]
### Summary

PyCharm will crash if I select a custom theme background with opacity. 

### Expected behavior

The background will change to the chosen color.

### Actual behavior

Some colors change, but the whole screen flickers, some fonts start to overlap, I can't exit PyCharm by exiting normally through file -> exit, only if I kill the process.

### Steps to reproduce (if needed)

1. Go to Settings -> Appearance & Background -> Material Custom Theme -> Background Color 

2. Choose a color with opacity.

(3. If you want to get Pycharm working again, kill it's process & the java process, delete the material_custom_theme.xml file and then start Pycharm.)

Here's the output of the material_custom_theme.xml file that lead to a broken Pycharm.

```
<application>
  <component name="MaterialCustomThemeConfig">
    <option name="excludedColor" value="2e3c43ff" />
    <option name="accentColor" value="009688ff" />
    <option name="notificationsColor" value="323232ff" />
    <option name="treeSelectionColor" value="546e7a50" />
    <option name="highlightColor" value="425b67ff" />
    <option name="secondBorderColor" value="2a373eff" />
    <option name="tableSelectedColor" value="314549ff" />
    <option name="contrastColor" value="1e272cff" />
    <option name="disabledColor" value="415967ff" />
    <option name="secondaryBackgroundColor" value="32424aff" />
    <option name="buttonColor" value="2e3c43ff" />
    <option name="selectionForegroundColor" value="ffffffff" />
    <option name="selectionBackgroundColor" value="546e7aff" />
    <option name="textColor" value="607d8bff" />
    <option name="foregroundColor" value="b0bec5ff" />
    <option name="backgroundColor" value="ff103565" />
  </component>
</application>
```

### Environment

PyCharm 2019.1.2 (Community Edition)
Build #PC-191.7141.48, built on May 7, 2019
JRE: 11.0.2+9-b159.56 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-47-generic

MaterialUI 3.9.3.1

material_theme.xml content:
```

<application>
  <component name="MaterialThemeConfig">
    <option name="isHighContrast" value="true" />
    <option name="isWizardShown" value="true" />
    <option name="monochromeIcons" value="true" />
    <option name="pristineConfig" value="false" />
    <option name="selectedTheme" value="CUSTOM" />
    <option name="userId" value="<###DELETED BY ME###>" />
    <option name="version" value="3.9.3.1" />
  </component>
</application>
```

**Comment by Fr-S** [comment]
material_theme.xml content:

<application>
  <component name="MaterialThemeConfig">
    <option name="isHighContrast" value="true" />
    <option name="isWizardShown" value="true" />
    <option name="monochromeIcons" value="true" />
    <option name="pristineConfig" value="false" />
    <option name="selectedTheme" value="CUSTOM" />
    <option name="userId" value="<###DELETED BY ME###>" />
    <option name="version" value="3.9.3.1" />
  </component>
</application>

**Comment by mallowigi** [comment]
I'll remove the opacity option for background, foreground, accent and excluded files as they can cause such issues.

**Comment by mallowigi** [comment]
Fixed in 3.9.4
